### PR TITLE
chore(Dockerfile): add health-check shell script

### PR DESCRIPTION
**Pull request by Simon-Tl**
## What this PR does / why we need it?

Add the health check script to confirm the status after startup more quickly

### What is the impact of this PR?

No Breaking Change

<details><summary>CI Settings</summary><br/>

### **CI Usage**

**Tip**: Check the CI you want to run below, and then comment `/run-ci`.

**CI Switch**

- [ ] E2E Tests
- [ ] Web3 Compatible Tests
- [ ] OCT 1-5 And 12-15
- [ ] OCT 6-10
- [ ] OCT 11
- [ ] OCT 16-19
- [ ] v3 Core Tests

### **CI Description**

| CI Name                                   | Description                                                               |
| ----------------------------------------- | ------------------------------------------------------------------------- |
| *Chaos CI*                                | Test the liveness and robustness of Axon under terrible network condition |
| *Cargo Clippy*                            | Run `cargo clippy --all --all-targets --all-features`                     |
| *Coverage Test*                           | Get the unit test coverage report                                         |
| *E2E Test*                                | Run end-to-end test to check interfaces                                   |
| *Code Format*                             | Run `cargo +nightly fmt --all -- --check` and `cargo sort -gwc`           |
| *Web3 Compatible Test*                    | Test the Web3 compatibility of Axon                                       |
| *v3 Core Test*                            | Run the compatibility tests provided by Uniswap V3                        |
| *OCT 1-5 \| 6-10 \| 11 \| 12-15 \| 16-19* | Run the compatibility tests provided by OpenZeppelin                      |

</details>

### CODE REVIEW

File: devtools/ci/scripts/health_check.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```
Comment (Flouse) [inline devtools/ci/scripts/health_check.sh]: It's better to put this script into `devtools/docker` dir.

File: devtools/ci/scripts/health_check.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+block_number() {
+    block=$(curl -s 'http://127.0.0.1:8000' \
+            --header 'Content-Type: application/json' \
+            --data '{"jsonrpc":"2.0", "method":"eth_blockNumber", "params": [], "id":42}' | jq '.result' |xargs printf %d 0xF)
+    echo $block
+}
+
+
+
+
```
Comment (Flouse) [inline devtools/ci/scripts/health_check.sh]:
```suggestion
```

File: devtools/ci/scripts/health_check.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        sleep 1
+    done
+
+    echo "block does not grow in one minute, please check"
+    return 1
+}
+
+
+block_stats
```
Comment (Flouse) [inline devtools/ci/scripts/health_check.sh]:
```suggestion
block_stats

```

**Review by Flouse — CHANGES_REQUESTED** [review]

File: devtools/ci/scripts/health_check.sh (hunk 1 above)
Comment (Simon-Tl) [reply devtools/ci/scripts/health_check.sh]: ok, let me revise it.

**Review by Flouse — APPROVED** [review]

**Review by jiangxianliang007 — APPROVED** [review]

**Pull request merged**